src/icons/css/sprite.css

```diff
@@ -0,0 +1,506 @@
+.hyicon {
+    font-size: 10px;
+}
+
+.hyicon:before {
+    content:' ';
+    vertical-align:middle;
+    display: inline-block;
+    background-image: url("../svg/sprite.svg");
+    background-repeat: no-repeat;
+    background-size: 13.2em 712.8em;
+}
+
+.no-svg .hyicon:before {
+    background-image: url("../svg/sprite.png");
```

Comment:
Currently, there is no option to disable the generation of this class. I have opened the issue with the plugin author git repo. We may have to just live with it for now.